transforms/helpers/parse.ts

```diff
@@ -0,0 +1,32 @@
+import type { ParserOptions } from '@babel/core';
+import { parse } from '@babel/parser';
+import type { Parser } from 'jscodeshift';
+
+// Inspired by https://github.com/ember-codemods/ember-component-template-colocation-migrator/blob/50c37e5ab8710ced7815dd6c968af97cade23aa4/lib/utils/js-parser.js#L7
+
+const options: ParserOptions = {
+  sourceType: 'module',
+  allowImportExportEverywhere: true,
+  tokens: true,
+  plugins: [
+    // Without this, our @classic decorators cause this error:
```

Comment:
Stage 3 decorators actually make this mandatory -- we don't necessarily need to deal with this right
now, but we should investigate.

Reply:
Interesting choice. I'm sure there's good reasoning behind it.

There are some cases where this codemod could spint out a good number of class decorators (e.g. `@classic`, `@layout`, `@tagName`, `@attributeBindings`, etc) so putting it between `export` and `class` would be kinda ugly IMO.